keysign/receive.py

```diff
@@ -46,7 +46,9 @@
 from .keyfprscan import KeyFprScanWidget
 from .keyconfirm import PreSignWidget
 from .gpgmh import openpgpkey_from_data
-from .util import sign_keydata_and_send
+from .util import sign_keydata_and_send, fix_infobar
+from .avahiwormholediscover import AvahiWormholeDiscover
+from twisted.internet import reactor
```

Comment:
Move this up to external imports. I think it's meant to be "stdlib, external lib, own libs".